modules/local/plot_run_gantt/nextflow.config

```diff
@@ -6,4 +6,11 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
-}
\ No newline at end of file
+}
+
+profiles {
+    docker            { process.withName: 'PLOT_RUN_GANTT' { container = 'community.wave.seqera.io/library/click_pandas_plotly_express_typing:21adb9e2d1b605a5' } }
+    docker_arm64      { process.withName: 'PLOT_RUN_GANTT' { container = 'community.wave.seqera.io/library/click_pandas_plotly_express_typing:2e5e17c7ed2d1115' } }
+    singularity       { process.withName: 'PLOT_RUN_GANTT' { container = 'oras://community.wave.seqera.io/library/click_pandas_plotly_express_typing:a4af841350996386' } }
+    singularity_arm64 { process.withName: 'PLOT_RUN_GANTT' { container = 'oras://community.wave.seqera.io/library/click_pandas_plotly_express_typing:3fe674b9fa7b15b8' } }
```

Comment:
We don't need withName selectors

Reply:
This file is being loaded by the top level pipeline, as part of the regular pipeline config. So I think we do?

Reply:
yeah, that's why I asked for module config, otherwise this is almost the same as proposal 4